openapi_python_client/parser/properties/__init__.py

```diff
@@ -250,13 +250,23 @@ def build_model_property(
     required_properties: List[Property] = []
     optional_properties: List[Property] = []
     relative_imports: Set[str] = set()
+    references: List[oai.Reference] = []
 
     class_name = data.title or name
     if parent_name:
         class_name = f"{utils.pascal_case(parent_name)}{utils.pascal_case(class_name)}"
     ref = Reference.from_ref(class_name)
 
-    for key, value in (data.properties or {}).items():
+    all_props = data.properties or {}
+    if not isinstance(data, oai.Reference) and data.allOf:
+        for sub_prop in data.allOf:
+            if isinstance(sub_prop, oai.Reference):
+                references += [sub_prop]
```

Comment:
```suggestion
                references.append(sub_prop)
```
I'm not completely sure on the internals of how append vs + works, but I think this is probably faster since += reassigns the array.